demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -3501,9 +3538,24 @@ export class DemoMeetingApp
     return tiles;
   }
 
-  setUpVideoTileElementResizer(): void {
+  setupVideoTiles(): void {
+    const tileArea = document.getElementById(`tile-area`);
     for (let i = 0; i <= DemoTileOrganizer.MAX_TILES; i++) {
-      const videoElem = document.getElementById(`video-${i}`) as HTMLVideoElement;
+      let tile = document.createElement('video-tile') as DemoVideoTile
```

Comment:
nit: const instead of let?

Reply:
Need to modify a couple lines below.

Reply:
You just set properties right? I think you can modify properties for const object :D